CHANGES.md

```diff
@@ -12,6 +12,7 @@ Release Notes.
 * The operation name of quartz-scheduler plugin, has been changed as the `quartz-scheduler/${className}` format.
 * Fix jdk-http and okhttp-3.x plugin did not overwrite the old trace header.
 * Support collecting logs of log4j, log4j2, and logback in the tracing context with a new `logger-plugin`.
+* Fix thrift plugin trace link broken when intermediate service does not mount agent, and collect wrong args when method without parameters.
```

Comment:
```suggestion
* Fix thrift plugin trace link broken when intermediate service does not mount agent
* Fix thrift plugin collects wrong args when the method without parameter.
```

Reply:
2lines of changelog seem more clear. Agree?

Reply:
👍